### ISSUE TITLE
[accessibility] filter buttons aren't keyboard operable

### ISSUE THREAD

**Issue by Malvoz** [maydemirx/leaflet-tag-filter-button#38]
It's currently not possible to use a keyboard to operate the filter tags.

Ideally `<button>` should be used and not `<a>`, please see the <a href="https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html"><span title="Web Accessibility Initiative - Accessible Rich Internet Applications">WAI-ARIA</span> recommendation</a>.

**Comment by Malvoz** [comment]
Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)

**Comment by maydemirx** [comment]
Thank you @Malvoz for your feedback. I'll check and inform again.
